app/lib/features/chat/widgets/member_list.dart

```diff
@@ -36,7 +43,7 @@ class MemberList extends ConsumerWidget {
               padding: const EdgeInsets.all(8.0),
               child: MemberListEntry(
                 member: member,
-                convo: convo,
+                convo: roomNotifier.asyncRoom.requireValue,
```

Comment:
I really feel that this roomNotifier.asyncRoom didn't change things to the better, considering the increased number of `requireValue` that it comes with...